.github/workflows/1-experiment.yaml

```
@@ -1,85 +1,75 @@
-name: train-model
-
+name: Train
 on:
   push:
-    branches-ignore: # exclude master and dev below
-      - main
-      - dev
-
+    # don't run on `main` and `dev` commits
+    branches-ignore: [main, dev]
 jobs:
   deploy-runner:
-    if: contains(github.event.head_commit.message, '[cml]')
     runs-on: ubuntu-latest
+    environment: cloud
+    permissions:
+      contents: read
+      id-token: write
     steps:
       - uses: actions/checkout@v2
       - uses: iterative/setup-cml@v1
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::342840881361:role/SandboxUser
       - name: deploy
-        shell: bash
         env:
-          repo_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }} # set your own via gh cli or UI
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }} # set your own via gh cli or UI
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} # set your own via gh cli or UI
-          AWS_SESSION_TOKEN: ${{ secrets.AWS_SESSION_TOKEN }} # set your own via gh cli or UI
+          # set these secrets via gh CLI or UI. See also:
+          # https://cml.dev/doc/self-hosted-runners?tab=GitHub#personal-access-token
+          REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
         run: |
-          cml-runner \
-          --cloud aws \
-          --cloud-region us-east-1 \
-          --cloud-type g4dn.xlarge \
-          --labels=cml-runner
+          cml runner \
+              --cloud=aws \
+              --cloud-region=us-east-1 \
+              --cloud-type=g4dn.xlarge \
+              --labels=cml-runner
   train-model:
-    if: contains(github.event.head_commit.message, '[cml]')
     needs: deploy-runner
     runs-on: [self-hosted, cml-runner]
     container:
-      image: docker://iterativeai/cml
+      image: docker://iterativeai/cml:0-dvc2-base1
       options: --gpus all
     steps:
       - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
       - uses: actions/setup-python@v2
         with:
           python-version: "3.9"
       - name: Setup Git User
-        run: |
-          # Get user name and email from previous commit
-          USER_NAME=$(git --no-pager log --format=format:'%ae' -n 1)
-          USER_EMAIL=$(git --no-pager log --format=format:'%ae' -n 1)
-          git config --local user.name "$USER_NAME"
-          git config --local user.email "$USER_EMAIL"
+        run: cml ci
```

```suggestion
        run: cml ci
        env:
          REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

:see_no_evil: fix error :)